### ISSUE TITLE
Lightpipeline bug when text contains slashes inside (e.g, /string/)

### ISSUE THREAD

**Issue by josejuanmartinez** [JohnSnowLabs/spark-nlp#13102]
Slashes are triggering a error in LP. At some point it thinks it's processing URL instead of raw texts and triggers a "Path must be absolute" error.

## Description

Hi! Very weird bug. If you use LightPipeline (see configuration below) in a text with a string having slashes inside, it gets bugged:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/36634572/200305525-4f170c19-ca18-4826-842a-b439da555e68.png">

If we remove `/` it works perfectly, so it's definitely the slashes.
<img width="707" alt="image" src="https://user-images.githubusercontent.com/36634572/200306389-88d6d202-9238-40b3-be9e-b4562e7ee1c3.png">

How to reproduce:

PIPELINE
```
def get_generic_base_pipeline():
  """Common components used in all pipelines"""
  document_assembler = nlp.DocumentAssembler()\
      .setInputCol("text")\
      .setOutputCol("document")

  sentence_detector = nlp.SentenceDetector()\
      .setInputCols(["document"])\
      .setOutputCol("sentence")

  tokenizer = nlp.Tokenizer()\
      .setInputCols(["sentence"])\
      .setOutputCol("token")

  embeddings = nlp.BertEmbeddings.pretrained("bert_embeddings_sec_bert_base","en") \
      .setInputCols(["sentence", "token"])\
      .setOutputCol("embeddings")

  base_pipeline = Pipeline(stages=[
      document_assembler,
      sentence_detector,
      tokenizer,
      embeddings
  ])

  return base_pipeline
    
generic_base_pipeline = get_generic_base_pipeline()

ner_model_role = finance.NerModel.pretrained("finner_org_per_role_date", "en", "finance/models")\
    .setInputCols(["sentence", "token", "embeddings"])\
    .setOutputCol("ner_role")

ner_converter_role = nlp.NerConverter()\
    .setInputCols(["sentence","token","ner_role"])\
    .setOutputCol("ner_chunk_role")

pos = PerceptronModel.pretrained()\
    .setInputCols(["sentence", "token"])\
    .setOutputCol("pos")

dependency_parser = DependencyParserModel().pretrained("dependency_conllu", "en")\
    .setInputCols(["sentence", "pos", "token"])\
    .setOutputCol("dependencies")

re_ner_chunk_filter_role = finance.RENerChunksFilter()\
    .setInputCols(["ner_chunk_role", "dependencies"])\
    .setOutputCol("re_ner_chunk_role")\
    .setRelationPairs(["PERSON-ROLE", "ORG-ROLE", "DATE-ROLE"])\
    .setMaxSyntacticDistance(5)

re_model_exp = finance.RelationExtractionDLModel.pretrained("finre_work_experience_md", "en", "finance/models")\
    .setInputCols(["re_ner_chunk_role", "sentence"])\
    .setOutputCol("relations")\
    .setPredictionThreshold(0.5)

nlpPipeline = Pipeline(stages=[
    generic_base_pipeline,
    ner_model_role,
    ner_converter_role,
    pos,
    dependency_parser,
    re_ner_chunk_filter_role,
    re_model_exp,
])


model = nlpPipeline.fit(empty_data)
light_model = LightPipeline(model)
```

TEXT:
```
n\n\nSIGNATURES\n\nPursuant to the requirements of Section 13 or 15(d) of the Securities Exchange Act of 1934, the registrant has duly caused this report to be signed on its behalf by the undersigned, thereunto duly authorized.\n                        \nCADENCE DESIGN SYSTEMS, INC.\n/s/ Anirudh Devgan\nAnirudh Devgan\nPresident and Chief Executive Officer\nDated: \nFebruary 22, 2022\n\n\nPursuant to the requirements of the Securities Exchange Act of 1934, this report has been signed below by the following persons on behalf of the registrant and in the capacities and on the dates indicated.\n/s/ Anirudh Devgan\nDATE: \nFebruary 22, 2022\nAnirudh Devgan\nPresident and Chief Executive Officer\n/s/ John M. Wall\nDATE: \nFebruary 22, 2022\nJohn M. Wall\nSenior Vice President and Chief Financial Officer\n\n\n86\n
```

## Expected Behavior

THe LightPipeline should be able to process a text without slashes instead of crashing.

## Current Behavior

It breaks the processing, triggering the error shown above.

## Possible Solution

I'm not sure why a raw text is processed as urls in LightPipeline, so I don't have a good solution. The bad solution is of course escaping.

## Steps to Reproduce

(attached above)

## Context

I try to run a LightPipeline on a text with slashes and I jus can't. I need to replace them.

## Your Environment

* Spark NLP version `sparknlp.version()`: 4.2.1 and 4.2.2
* Apache NLP version `spark.version`: 3.1.2
* Java version `java -version`: 1.8
* Setup and installation (Pypi, Conda, Maven, etc.): Pypi
* Operating System and version: Ubuntu 18
* Link to your project (if any): code attached above

**Comment by maziyarpanahi** [comment]
@josejuanmartinez we have done some refactoring for LightPipeline in `4.2.2` release, would it be possible to test it with 4.2.2 release?

**Comment by josejuanmartinez** [comment]
@maziyarpanahi It still happens in 4.2.2.

<img width="1212" alt="image" src="https://user-images.githubusercontent.com/36634572/202440370-78caac82-b8c1-446e-9f00-5f14471ec6d2.png">

**Comment by maziyarpanahi** [comment]
Thanks @josejuanmartinez 
Could you please copy/paste the full error trace? It's not visible where and why.

**Comment by maziyarpanahi** [comment]
Thanks @josejuanmartinez 

@danilojsl it seems detecting `path` via `/` is not safe in `__isPath` method. I have a Colab here, tried to reproduce it but I couldn't: https://colab.research.google.com/drive/1QP3f0oP0lUwlvS4VPm6QjsOA7Xrt5MN-?usp=sharing

But it is clearly possible to mistaken the text input as image path.

**Comment by josejuanmartinez** [comment]
Check if it's `hdfs` related. Here is the notebook to reproduce it - just run all cells.
https://adb-7956323724731612.12.azuredatabricks.net/?o=7956323724731612#notebook/281510005229057/command/281510005229058

**Comment by maziyarpanahi** [comment]
@danilojsl I sent you an invite to Databricks on Azure so you can access that notebook

**Comment by danilojsl** [comment]
Hi @maziyarpanahi, @josejuanmartinez . I remember I thought of URL as an input edge case when implementing LightPipeline, and made the required changes, but it looks like I missed something. I will check the bug and come up with a solution.

**Comment by ar9** [comment]
@danilojsl my LP does not work when my data has dates in the mm/dd/yy format, e.g. 11/18/22

**Comment by danilojsl** [comment]
> @danilojsl my LP does not work when my data has dates in the mm/dd/yy format, e.g. 11/18/22

@ar9 could you please provide the full pipeline to reproduce the error? I did a quick example and I cannot reproduce the issue:

```python
pipeline.fullAnnotate("11/18/22")
```
Output:

```
[{'entities': [],
  'stem': [Annotation(token, 0, 7, 11/18/22, {'confidence': '0.0', 'sentence': '0'})],
  'checked': [Annotation(token, 0, 7, 11/18/22, {'confidence': '0.0', 'sentence': '0'})],
  'lemma': [Annotation(token, 0, 7, 11/18/22, {'confidence': '0.0', 'sentence': '0'})],
  'document': [Annotation(document, 0, 7, 11/18/22, {})],
  'pos': [Annotation(pos, 0, 7, CD, {'word': '11/18/22', 'sentence': '0'})],
  'token': [Annotation(token, 0, 7, 11/18/22, {'sentence': '0'})],
  'ner': [Annotation(named_entity, 0, 7, O, {'word': '11/18/22', 'sentence': '0'})],
  'embeddings': [Annotation(word_embeddings, 0, 7, 11/18/22, {'isOOV': 'true', 'pieceId': '-1', 'isWordStart': 'true', 'token': '11/18/22', 'sentence': '0'})],
  'sentence': [Annotation(document, 0, 7, 11/18/22, {'sentence': '0'})]}]
```

**Comment by ar9** [comment]
@danilojsl sorry, please disregard. My tokenizer did not include the forward slash as a suffix.

**Comment by danilojsl** [comment]
@josejuanmartinez I reproduce the error also using only an open-source pipeline with spark-nlp 4.2.0 https://adb-7956323724731612.12.azuredatabricks.net/?o=7956323724731612#notebook/646932875973418/command/646932875973424. However, with 4.2.1 it works. Are you sure you tested using a cluster with 4.2.1?

**Comment by josejuanmartinez** [comment]
I thought I tested it with 4.2.1 and 4.2.2, but I will check again. Thank you @danilojsl

**Comment by maziyarpanahi** [comment]
@josejuanmartinez Can you confirm that the issue was/is either:
- The bad licensed models are not correctly being uploaded (some of them were compressed in macOS where the .zip root is inside another zip folder if it is compressed via UI)
- The spark-nlp version was 4.2.0 and not 4.2.1 and above?

Just need to conclude this issue for the next release

**Comment by josejuanmartinez** [comment]
Hi Maz. My guess is that it was DB environment not taking  4.2.2, and taking earlier versions. I did change it and restarted, but I may have missed something. 

So if you want to announce the bug fix, basically it should be something like... `A bug triggered by having slashes in texts processed with LightPipelines has been fixed in >= 4.2.2`

**Comment by maziyarpanahi** [comment]
> Hi Maz. My guess is that it was DB environment not taking 4.2.2, and taking earlier versions. I did change it and restarted, but I may have missed something.
> 
> So if you want to announce the bug fix, basically it should be something like... `A bug triggered by having slashes in texts processed with LightPipelines has been fixed in >= 4.2.2`

@josejuanmartinez thanks a lot! This is great and we will take note of this if something similar has come up. 🙏
